db/changelog/v-1/event_scheduler.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="event-scheduler-1" author="frolov" context="prod">
```

Comment:
We should think about a way to keep some sort of execution history even after we remove test executions, maybe store list of failed tests in Execution? Anyway, it's a thought for a separate issue

Reply:
Yes I agree. We need to create a issue for this.

Reply:
#69